apps/web/src/app/api/v1/workspaces/[wsId]/integrations/sepay/sepay-integration.md

```
@@ -0,0 +1,288 @@
+# SePay Webhook + Auto-Categorization Implementation Plan
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)